recipes/libnvshmem/meta.yaml

```
@@ -0,0 +1,225 @@
+        - cudatoolkit       # [(cuda_compiler_version or "").startswith("11")]
+        - libpciaccess
+      run:
+        - {{ pin_compatible("cuda-version", min_pin="x", max_pin="x") }}
+        - {{ pin_subpackage("libnvshmem" ~ soname, exact=True) }}
+        - cudatoolkit       # [(cuda_compiler_version or "").startswith("11")]
+        - cuda-cudart       # [(cuda_compiler_version or "").startswith("12")]
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+    test:
+      commands:
+        - test -f $PREFIX/bin/hydra_pmi_proxy
+        - test -f $PREFIX/bin/nvshmem-info
+        - test -f $PREFIX/bin/nvshmrun.hydra
+        - test -f $PREFIX/bin/hydra_nameserver
+        - test -f $PREFIX/bin/nvshmrun
+        - test -f $PREFIX/bin/hydra_persist
+        - test -f $PREFIX/bin/examples/collective-launch
+        - test -f $PREFIX/bin/examples/on-stream
+        - test -f $PREFIX/bin/perftest/device/pt-to-pt/shmem_p_latency
+        - test -f $PREFIX/bin/perftest/device/coll/reduction_latency
+        - test -f $PREFIX/bin/perftest/host/init/malloc
+        - test -f $PREFIX/bin/perftest/host/pt-to-pt/bw
+        - test -f $PREFIX/bin/perftest/host/coll/broadcast_on_stream
+        - test -f $PREFIX/include/bootstrap_device_host/nvshmem_uniqueid.h
+        - test -f $PREFIX/include/device/nvshmem_defines.h
+        - test -f $PREFIX/include/device_host/nvshmem_types.h
+        - test -f $PREFIX/include/host/nvshmem_api.h
+        - test -f $PREFIX/include/non_abi/device/coll/defines.cuh
+        - test -f $PREFIX/include/device_host_transport/nvshmem_constants.h
+        - test -f $PREFIX/include/nvshmem.h
+        - test -f $PREFIX/include/nvshmemx.h
+        - test -f $PREFIX/lib/cmake/nvshmem/NVSHMEMConfig.cmake
+        - test -f $PREFIX/lib/cmake/nvshmem/NVSHMEMDeviceTargets-release.cmake
+        - test -f $PREFIX/lib/cmake/nvshmem/NVSHMEMDeviceTargets.cmake
+        - test -f $PREFIX/lib/cmake/nvshmem/NVSHMEMTargets-release.cmake
+        - test -f $PREFIX/lib/cmake/nvshmem/NVSHMEMTargets.cmake
+        - test -f $PREFIX/lib/cmake/nvshmem/NVSHMEMVersion.cmake
+        - test -L $PREFIX/lib/libnvshmem_host.so
+        - test -L $PREFIX/lib/nvshmem_bootstrap_mpi.so
+        - test -L $PREFIX/lib/nvshmem_bootstrap_pmi.so
+        - test -L $PREFIX/lib/nvshmem_bootstrap_pmi2.so
+        - test -L $PREFIX/lib/nvshmem_bootstrap_pmix.so
+        - test -L $PREFIX/lib/nvshmem_bootstrap_shmem.so
+        - test -L $PREFIX/lib/nvshmem_bootstrap_uid.so
+        - test -L $PREFIX/lib/nvshmem_transport_ibdevx.so
+        - test -L $PREFIX/lib/nvshmem_transport_ibgda.so
+        - test -L $PREFIX/lib/nvshmem_transport_ibrc.so
+        - test -L $PREFIX/lib/nvshmem_transport_libfabric.so
+        - test -L $PREFIX/lib/nvshmem_transport_ucx.so
+
+  - name: libnvshmem-static
+    build:
+      script: |
+        cp -rv lib/libnvshmem*.a $PREFIX/lib/
+    files:
+      - lib/libnvshmem*.a
```

```suggestion
      script: |
        cp -rv lib/libnvshmem*.a $PREFIX/lib/
```
See other comment about possibly needing to make $PREFIX/lib before you try to copy into it.

---

This will be replaced with global `build.sh`